### PR TITLE
feat: add mocked mode in dev/test

### PR DESCRIPTION
**Pull request by Roguelazer**

### CODE REVIEW

File: lib/rails_cloudflare_turnstile/view_helpers.rb [hunk 1]
```diff
@@ -25,6 +30,15 @@ def turnstile_div(action)
       end
     end
 
+    def mock_turnstile_div(action)
+      content_tag(:div, class: "cf-turnstile", style: "width: 300px; height: 65px: border: 1px solid gray") do
```
Comment (gsinkin-instrumentl) [inline lib/rails_cloudflare_turnstile/view_helpers.rb:34]: we should throw a `class` on this baddy, rather than `style`

Comment (Roguelazer) [reply lib/rails_cloudflare_turnstile/view_helpers.rb:34]: I don't actually know how to inject CSS from a rails plugin into the main app. I assume there's some way to provide a SASS file that goes into the build flow?

Comment (gsinkin-instrumentl) [reply lib/rails_cloudflare_turnstile/view_helpers.rb:34]: Ah, yea, this is an oddball here. Hmmmmmm... we could use a `stylesheet_link_tag`, but this is feeling overkill-y.

Comment (Roguelazer) [reply lib/rails_cloudflare_turnstile/view_helpers.rb:34]: yeah I think for now maybe we just leave the style tag on it?

**Review by gsinkin-instrumentl — APPROVED** [review]

**Pull request merged**